### ISSUE TITLE
Read Azure configuration from env variables

### ISSUE THREAD

**Issue by tommasobertoni** [artiso-solutions/CoVoX#43]
Provide additional settings API to read the azure keys from environment variables.
like:
```
COVOX__AZURE__SUBSCRIPTIONKEY
COVOX__AZURE__REGION
```
or
```
COVOX__AZURE__CONFIG (subscriptionKey:region)
```

**Comment by antonio8101** [comment]
I think that it could be better to have 2 env variables not one that should be splitted used magic words.. That's to make simpler the troubleshooting when something it's not correct.. 
I would suggest to have two key env `COVOX_AZURE_SUBSCRIPTION`, `COVOX_AZURE_REGION`

**Comment by kczornik** [comment]
I pushed the changes to the [proposals/ensure-valid-configuration](https://github.com/artiso-solutions/CoVoX/tree/proposals/ensure-valid-configuration) branch. Changed can be merged in #51
